examples/job/generate-slabs-al2o3.py

```diff
@@ -0,0 +1,250 @@
+#!/usr/bin/env python
```

Comment:
We should have a folder named `assets` inside the `examples` or even at the top level and utilize it for static files like poscars

Reply:
Refactored poscars into a new assets folder